### PR TITLE
feat(footer):added a footer with links

### LINKED ISSUE #173: em-footer
Description
Add a simple footer to the page

Tasks
Include specific tasks in the order they need to be done in. Include links to specific lines of code where the task should happen at.

[ ] Add footer

### PR DESCRIPTION
**Pull request by Eddie-Moreno**
#Changes
List Changes Introduced by this PR
*Added a footer
*Links

#Purpose
footer with links and contact

#If this closes an issue, reference the issue here. If it doesn't, remove this line 
Closes #173

### CODE REVIEW

**Review by jacquelinemartinez6114 — APPROVED** [review]
Footer seems good and Worked with Eddie on a bug on the footer, it seems to be fixed. Good

**Review by Kendallfuller97 — APPROVED** [review]
Nice

File: Eddie/styles.css [hunk 1]
```diff
@@ -81,7 +81,7 @@
     display: flex;
     justify-content: center;
   
-    width: 98vw;
+    width: 100%;
```
Comment (gabymoreno9) [inline Eddie/styles.css:84]: Example: does this make sense?

**Review by gabymoreno9 — COMMENTED** [review]
Looks good overall, I just had a comment on one of the lines of code.

File: Eddie/Eddie.html [hunk 2]
```diff
@@ -60,6 +62,31 @@ <h2 class="about-me">About Me</h2>
     
 
     <!-- Main Content End -->
+
+    <!-- Footer start -->
+    <div class="text-color text-center">
+      <footer>
```
Comment (gabymoreno9) [inline Eddie/Eddie.html:68]: Example: Great looking footer!

File: Eddie/Eddie.html [hunk 3]
```diff
@@ -60,6 +62,31 @@ <h2 class="about-me">About Me</h2>
...
+            <h3>Connect with me</h3>
+            <a href="https://github.com/eddie-moreno" class="fa-brands fa-github"></a>
+            <a href="https://www.linkedin.com/" class="fa-brands fa-linkedin"></a>
+            <div class="copyright mt-5">
+              <p>&copy;Eddie L. Moreno 2022</p>
+            </div>
+          </div>
+          <div class="col">
+            <h4>Email:</h4>
+            <p><a class="email" href="mailto:eddiemoreno@gmail.com">eddiemoreno@gmail.com</a></p>
```
Comment (gabymoreno9) [inline Eddie/Eddie.html:84]: example: typo here (not really using this as an example)

**Review by gabymoreno9 — COMMENTED** [review]
Overall great job, I had a few questions (example)

**Pull request merged**